infoschema/tables.go

```diff
@@ -646,6 +649,16 @@ var tableTiKVRegionPeersCols = []columnInfo{
 	{"DOWN_SECONDS", mysql.TypeLonglong, 21, 0, 0, nil},
 }
 
+var tableTiDBServersInfoCols = []columnInfo{
+	{"DDL_ID", mysql.TypeVarchar, 64, 0, nil, nil},
```

Comment:
`DDL_ID` is not so important in server info, I think it should be placed after `STATUS_PORT`.
BTW, column name `DDL_ID` makes users confused, is there any better name?

Reply:
`DDL_ID` is important when we want to know which `tidb-server` is the owner. 
But the value actually is the DDL id.